scripts/python/singlepss.py

```diff
@@ -203,9 +209,15 @@ def buf_get(pssName, typ, name, known):
 			shortname = "pss:" + name[:8]
 			ispubkey = True
 
-		buf = weechat.buffer_new(bufname, "buf_in", pssName, "buf_close", pssName)
+		buf = weechat.buffer_new(bufname, "buf_in", pssName, "buf_close",
+		                         pssName)
 		weechat.buffer_set(buf, "short_name", shortname)
-		weechat.buffer_set(buf, "title", name + " @ PSS '" + pssName + "' | node: " + weechat.config_get_plugin(psses[pssName].host + "_url") + ":" + weechat.config_get_plugin(psses[pssName].port + "_port") + " | key  " + pss.label(psses[pssName].key) + " | address " + pss.label(psses[pssName].base))
+		weechat.buffer_set(
+		    buf, "title", name + " @ PSS '" + pssName + "' | node: " +
```

Comment:
yes this is messy. In these cases maybe build long string separately then put in call.